lib/Content/View/Builder/ContentEditViewBuilder.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\RepositoryForms\Content\View\Builder;
+
+use eZ\Publish\API\Repository\Repository;
+use eZ\Publish\Core\Base\Exceptions\InvalidArgumentException;
+use eZ\Publish\Core\MVC\Symfony\View\Builder\ViewBuilder;
+use eZ\Publish\Core\MVC\Symfony\View\Configurator;
+use eZ\Publish\Core\MVC\Symfony\View\ParametersInjector;
+use EzSystems\RepositoryForms\Content\View\ContentEditView;
+
+/**
+ * Builds ContentView objects.
+ *
+ * @internal
+ */
+class ContentEditViewBuilder implements ViewBuilder
+{
+    /** @var \eZ\Publish\API\Repository\Repository */
+    private $repository;
+
+    /** @var \eZ\Publish\Core\MVC\Symfony\View\Configurator */
+    private $viewConfigurator;
+
+    /** @var \eZ\Publish\Core\MVC\Symfony\View\ParametersInjector */
+    private $viewParametersInjector;
+
+    /** @var string */
+    private $defaultTemplate;
+
+    public function __construct(
+        Repository $repository,
+        Configurator $viewConfigurator,
+        ParametersInjector $viewParametersInjector,
+        $defaultTemplate
+    ) {
+        $this->repository = $repository;
+        $this->viewConfigurator = $viewConfigurator;
+        $this->viewParametersInjector = $viewParametersInjector;
+        $this->defaultTemplate = $defaultTemplate;
+    }
+
+    public function matches($argument)
+    {
+        return 'ez_content_edit:editVersionDraftAction' === $argument;
+    }
+
+    /**
+     * @param array $parameters
+     *
+     * @return \eZ\Publish\Core\MVC\Symfony\View\ContentView|\eZ\Publish\Core\MVC\Symfony\View\View
+     *
+     * @throws \eZ\Publish\Core\Base\Exceptions\InvalidArgumentException If both contentId and locationId parameters are missing
+     * @throws \eZ\Publish\Core\Base\Exceptions\UnauthorizedException
```

Comment:
Please type hint API exceptions when possible.
The method also throws `\eZ\Publish\API\Repository\Exceptions\NotFoundException`.